package.json

```diff
@@ -14,14 +14,14 @@
     "datastore"
   ],
   "dependencies": {
-    "async": "^2.6.1",
     "@azure/storage-blob": "^10.3.0",
+    "async": "^2.6.1",
     "azure-storage": "^2.10.2",
     "datastore-core": "~0.6.0",
-    "fs": "0.0.1-security",
     "interface-datastore": "~0.6.0",
     "pull-defer": "~0.2.3",
     "pull-stream": "^3.6.9",
+    "stream": "0.0.2",
```

Comment:
NPM page documentation says that this is a browser port from Node, so no need for this package?